### DISCUSSION TITLE
Unable to view the data of PMPOSHAN and NAS program on UAT server

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#844] [General]
Hi @RNarmatha  ,
I tried to update the recent data on PY-UAT server. After data ingestion process I am not able to view the PMPOSHAN and NAS program data on server. It shows 0 in server. But in Minio console the file has data. Help us to resolve this. Attaching the screenshot here-
![image](https://github.com/Sunbird-cQube/community/assets/130358614/9d826db1-2bef-436b-b7cb-d51ad5cec1e4)

**Comment by RNarmatha** [comment]
Rajesh could you please look into this issue

*Regards*





*Narmatha R*

Project Manager


*M* +91 9600095667

*E* ***@***.*** ***@***.***>*

[image: LinkedIn] <https://www.linkedin.com/company/1913934> [image:
Facebook] <https://www.facebook.com/TibilSolutions> [image: Twitter]
<https://twitter.com/TibilSolutions> [image: Insta]
<https://www.instagram.com/tibilsolutions/>
*This message (including any attachments) may contain confidential,
proprietary, privileged, and/or private information. The information is
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the
sender immediately, and delete the message and any attachments. Any
disclosure, reproduction, distribution, or other use of this message or any
attachments by an individual or entity other than the intended recipient is
prohibited.**t is prohibited.*

**Comment by RNarmatha** [comment]
Please check all the grammar files for those two programs are ingested

*Regards*





*Narmatha R*

Project Manager


*M* +91 9600095667

*E* ***@***.*** ***@***.***>*

[image: LinkedIn] <https://www.linkedin.com/company/1913934> [image:
Facebook] <https://www.facebook.com/TibilSolutions> [image: Twitter]
<https://twitter.com/TibilSolutions> [image: Insta]
<https://www.instagram.com/tibilsolutions/>
*This message (including any attachments) may contain confidential,
proprietary, privileged, and/or private information. The information is
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the
sender immediately, and delete the message and any attachments. Any
disclosure, reproduction, distribution, or other use of this message or any
attachments by an individual or entity other than the intended recipient is
prohibited.**t is prohibited.*

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Please confirm me which version of cQube are you installed ?

**Reply by Mohanapriya777** [reply]
V5.0.3

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Please check pm_poshan and nas data columns if you miss any thing it's not ingested. please find the below template and check the all the columns

https://docs.google.com/spreadsheets/d/10yWmivrqtxdRLiJpX_ytzxbvoGG0WNWEI-7RLV8zelM/edit#gid=886375954

**Reply by Mohanapriya777** [reply]
@bommalla @RNarmatha  Now i am able to view the data. Rearranged some columns. Thank you.

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Thanks for the update!!

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Thanks for the update!!

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Thanks for the update!!

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Thanks for the update!!
